e2e-execute.sh

```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
+shadowJar() {
+  echo "Running shadowJar..."
+  ./gradlew shadowJar
```

Comment:
```suggestion
  ./gradlew clean shadowJar
```

Reply:
I tried the `./gradlew clean shadowJar` but it didn't work. It seems to work with `./gradlew :app:clean shadowJar`

Reply:
Nevermind, it was a typo; `./gradlew clean shadowJar` works!